server/controller/central_checkers_controller.go

```diff
@@ -0,0 +1,92 @@
+package controller
+
+import (
+	"context"
+)
+
+// Tile int
+type Tile int
+
+// Tile enumeration
+const (
+	NoPiece Tile = iota
+	White
+	Black
+	WhitePromoted
+	BlackPromoted
+)
+
+// Move message
+type Move struct {
+	IndexFrom uint32
+	IndexTo   uint32
+}
+
+// BoardState message
+type BoardState struct {
+	Board []Tile
+}
+
+// BoardSubscriptionRequest message
+type BoardSubscriptionRequest struct {
+	GameID string
+}
+
+// MoveRequest message
+type MoveRequest struct {
+	MoveSet []Move
+}
+
+// MoveResponse message
+type MoveResponse struct {
+	MoveSuccess bool
+	Message     string
+}
+
+// BoardUpdate message
+type BoardUpdate struct {
+	BoardState  BoardState
+	PrevMoveSet []Move
+}
+
+// GameRole int
+type GameRole int
+
+// GameRole enumeration
+const (
+	PlayerWhite GameRole = iota
+	PlayerBlack
+	Spectator
+)
+
+// GameStartRequest message
+type GameStartRequest struct {
+	GameID string
+}
+
+// GameStartRole message
+type GameStartRole struct {
+	PlayerRole GameRole
+	GameID     string
```

Comment:
Is offsetting the types like this a good style?  I don't personally like it if the linter didn't suggest/require it.

Reply:
It is built into the golang linter and gofmt.